src/sonic-frr/patch/0006-BFD-Enhancement.patch

```diff
@@ -0,0 +1,1562 @@
+diff --git a/bfdd/bfd.c b/bfdd/bfd.c
```

Comment:
is this patch upstreamed?

Reply:
I have submitted Pull request in FRR to upstream these chages.